src/core/init.js

```diff
@@ -14,6 +14,15 @@ import { initialize as initTranslator } from './internationalization';
  * @return {Undefined}
  */
 const _globalInit = () => {
+  // Could have been any property defined within the p5 constructor.
+  // If that property is already a part of the global object,
+  // this code has already run before, likely due to a duplicate import
+  if (typeof window._setupDone !== 'undefined') {
```

Comment:
Why is it checking for `_setupDone` instead of `p5`? If using instance mode only `p5` will be attached to the window object and nothing else.

Reply:
@limzykenneth I found that window will have p5 attached to it even before this initialisation code runs. 

Here's what I get when I log `window.p5` at line 19 in the file
```
ƒ p5(sketch, node, sync) {
              var _this = this;
              _classCallCheck(this, p5);
              //////////////////////////////////////////////
              // PUBLIC p5 PROPERTIES AN…
```
The constructor is already attached to the window object. This is even with single import

Reply:
Yeah that's right, but with checking anything else that is on the `window` object means that it will only work for global mode and not instance mode.

Reply:
I'm not sure instance mode is a necessary consideration but it would be great if you can test it out.

Reply:
@limzykenneth it seems that instance mode is not affected by multiple imports, as it is up to the user to initialise p5 with `new p5(sketch)`. And the users can intentionally create as many instances as they want. When the browser concatenates the scripts to ready them for execution, I think the newer p5 stuff in this case simply replaces the old one with no other side effects.

The trouble in global mode is that the library itself takes the job of doing the `new p5()` initialisation, but with a duplicate import as the library itself is duplicated, this step happens twice. In this case, when the browser concatenates the scripts, `new p5()` is called for a second time for the second import. Hence, the user-defined setup() in sketch.js is called two times, which leads to the two canvases and other weird stuff.

Though yes, the method in this PR will not warn a user about multiple imports if instance mode is used. But as multiple imports here don't seem to break anything, is a warning necessary?

Reply:
Moreover, I think for instance mode, the state of the global variable will be the same for single or multiple imports so I am not really sure how to catch it.

Reply:
If it doesn't affect the behaviour of instance mode then I guess that's fine. Thanks.